archive_extractor/zip_archiver.go

```diff
@@ -122,3 +136,7 @@ func initZipReader(r io.ReaderAt, size int64) (*zip.Reader, error) {
 	}
 	return nil, errors.New("No zip file found")
 }
+
+func isZoneInfoFile(buf []byte) bool {
```

Comment:
let's have 2 sentences here to explain what is zoneInfo and why we need to identify it separately.